lib/diagnostic.rb

```diff
@@ -17,8 +21,12 @@
 # Finally, assign the modified Person to `Answer.dave` below.
 
 ##
-# your answers here
-Answer.dave = nil
+def initialize("Dave", 32, "Ohio")
+  @name = name
+  @age = age
+  @location = "Somerville"
+end
+Answer.dave = @location
```

Comment:
Sorry, this isn't right. We wanted you to instantiate a new Person. That means calling `.new`. 

```ruby
dave = Person.new("Dave", 32, "Ohio")
dave.location = "Somerville"
Answer.dave = dave
```